src/main/java/io/supertokens/storageLayer/StorageLayer.java

```diff
@@ -175,7 +182,7 @@ private static StorageLayer getInstance(TenantIdentifier tenantIdentifier, Main
     public static void initPrimary(Main main, String pluginFolderPath, JsonObject configJson)
             throws MalformedURLException, InvalidConfigException {
         main.getResourceDistributor().setResource(new TenantIdentifier(null, null, null), RESOURCE_KEY,
-                new StorageLayer(main, pluginFolderPath, configJson));
```

Comment:
what happens when you start the core with a tenant such that that tenant's db is down?